### ISSUE TITLE
Subtitle

### ISSUE THREAD

**Issue by farishsyazwan** [iv-org/invidious#2763]
https://user-images.githubusercontent.com/95562040/148147084-cce0ffbc-14e3-49fe-8423-9fd27f4d643e.mp4

I enabled caption as you can see above. I hope you can fix this. Thank you.

**Comment by SamantazFox** [comment]
I'm not sure what the problem you're reporting is?

**Comment by farishsyazwan** [comment]
> 

I enables subtitle but subtitle won't appear.

**Comment by unixfox** [comment]
Which instance are you using?

**Comment by farishsyazwan** [comment]
> Which instance are you using?

yewtu.be. Is this instance problem. Which instance that has working subtitle.

**Comment by unixfox** [comment]
Can you give us the video ID that you used?

**Comment by unixfox** [comment]
Okay I found the issue, the captions/subtitles are blocked by Google.

Duplicate of https://github.com/iv-org/invidious/issues/2565
